cpp/src/gandiva/llvm_generator_test.cc

```diff
@@ -115,4 +117,14 @@ TEST_F(TestLLVMGenerator, TestAdd) {
   EXPECT_EQ(out_bitmap, 0ULL);
 }
 
+TEST_F(TestLLVMGenerator, VerifyExtendedPCFunctions) {
+  ARROW_EXPECT_OK(LoadTestLLVMIR());
```

Comment:
Will this leak in other tests?

Reply:
Yes, currently we don't provide an API to unregister/unload a function, and the IR store is a global state which will affect other tests as well, but it should not cause any trouble for other tests.

Reply:
After making LLVMGenerator to accept a function registry, which itself is instantiable, now this test is updated and it will be independent with other tests.